ledger/src/proofs/wrap.rs

```diff
@@ -1903,8 +1908,8 @@ pub mod wrap_verifier {
                     (x, y)
                 })
                 .reduce(|mut acc, v| {
-                    acc.0 += &v.0;
-                    acc.1 += &v.1;
+                    acc.0 = Affine::from(acc.0 + v.0);
+                    acc.1 = Affine::from(acc.1 + v.1);
```

Comment:
Or you can compute everything in projective and convert back to affine afterwards, depending on how much data you process.